circle.yml

```diff
@@ -66,7 +66,9 @@ fedora_prep_cache: &fedora_prep_cache
   steps:
     - run:
         name: Initializing Fedora dnf cache
-        command: dnf install -y --downloadonly libsolv tree git gcc gcc-c++ meson check-devel libudev-devel libevdev-devel valgrind python3-gobject python3-evdev glib2-devel python3-lxml
+        command: |
+          dnf install -y --downloadonly libsolv tree git gcc gcc-c++ python3 check-devel libudev-devel libevdev-devel valgrind python3-gobject python3-evdev glib2-devel python3-lxml ninja-build
+          pip3 install meson
```

Comment:
nope, not a fan of this. I hate having to use pip on distributions, especially those reasonably up-to-date because it'll also force all our users to run pip to build libratbag. We have enough issues with the python code as it is, we don't need to make this harder.

This would also prevent distributions shipping libratbag (pip install is a no-go in fedora's build system) which again is something I don't like because it makes getting and installing libratbag harder.